pkg/controller/rest/verifiable/operation.go

```diff
@@ -155,23 +159,14 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 	rest.Execute(o.command.GeneratePresentation, rw, req.Body)
 }
 
-// GeneratePresentationByID swagger:route GET /verifiable/credential/{id}/presentation verifiable presentationByIDReq
+// GeneratePresentationByID swagger:route POST /verifiable/presentation/id/generate
```

Comment:
add a TODO or fix this endpoint. `/verifiable/presentation/generatebyid`

Reply:
fix rather than TODO.

Reply:
fixed